docs/HyperIndexV2/migration-guide.md

```
@@ -0,0 +1,252 @@
+    id: event.transaction.hash + event.logIndex,
+    tokenId: event.params.tokenId,
+    address: event.params.subscriber,
+    blockNumber: event.block.number,
+    logIndex: event.logIndex,
+    position: event.params.tokenId
+  }
+
+context.Subscription.set(entity);
+})
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)